### PR TITLE
Migrate to SASS math.div

### PR DESCRIPTION
**Pull request by rak-phillip**
This removes SASS deprecation warnings by migrating to `math.div`. In order to make use of `@use 'sass:math'` inside of our project, we first needed to update `@nuxtjs/style-resources`, which exposes the `hoistUseStatements` to inside of `nuxt.config`. After making these changes, we get access to `math.div` throughout our project.

#593

### CODE REVIEW

File: src/nuxt.config.js [hunk 1]
```diff
@@ -67,6 +68,7 @@ export default {
   ssr:            false,
   styleResources: {
     // only import functions, mixins, or variables, NEVER import full styles https://github.com/nuxt-community/style-resources-module#warning
+    hoistUseStatements: true,
     scss: [
```
Comment (mook-as) [inline src/nuxt.config.js:72]: Nit: lint warning (it wants the `[` to be more indented to match the `true` in the line above).

File: src/nuxt.config.js [hunk 2]
```diff
@@ -38,6 +38,7 @@ export default {
     '@nuxtjs/router-extras',
     '@nuxtjs/style-resources',
     '@nuxt/typescript-build',
+    '@nuxtjs/style-resources'
```
Comment (mook-as) [inline src/nuxt.config.js:41]: This was already there (the second element); do we need it again?

**Review by mook-as — CHANGES_REQUESTED** [review]
- Please use npm v7 (lockfile version 2) to reduce churn in `package-lock.json`.
- Please make a PR on [dashboard](https://github.com/rancher/dashboard) to ensure that the next time we pull in their files we will not need to manually resolve conflicts.

Thanks!

File: src/nuxt.config.js (hunk 2 above)
Comment (rak-phillip) [reply src/nuxt.config.js:41]: Nice catch - no clue why I did that

**Comment by rak-phillip** [comment]
@mook-as thanks for the heads up on npm v7 - I generally use the version of npm that comes bundled with node LTS. I'll be raising another PR in rancher dashboard.

**Review by mook-as — APPROVED** [review]
👍 Small diff, works as expected.

**Pull request merged**